pkg/cmd/repo/list/list.go

```diff
@@ -207,8 +211,6 @@ func listHeader(owner string, matchCount, totalMatchCount int, hasFilters bool)
 	if totalMatchCount == 0 {
 		if hasFilters {
 			return "No results match your search"
-		} else if owner != "" {
-			return "There are no repositories in @" + owner
```

Comment:
Let's bring this message back. It's useful as an error message to say `There are no repositories in @foo` if `foo` is a valid user or org